src/components/SearchBar/SearchBar.vue

```diff
@@ -16,7 +16,10 @@
       </template>
       <template #iconRight>
         <button
-          class="block"
+          :class="[
+            'block opacity-0 pointer',
```

Comment:
pointer is not a class?
also lets !opacity-100 to make sure it will override?
or something like `searchTerm ? 'opacity-100 cursor-pointer' : 'opacity-0'`

Reply:
You're right, its not!  I'll update this